cmd/etrace/main.go

```diff
@@ -271,11 +276,7 @@ func (x *cmdRun) Execute(args []string) error {
 		start := time.Now()
 		err = cmd.Start()
 
-		if x.NoWindowWait {
-			// if we aren't waiting on the window class, then just wait for the
-			// command to return
-			cmd.Wait()
-		} else {
+		if !x.NoWindowWait {
```

Comment:
hmm this is a bit awkward to read, but I think I can clean it up a bit in a followup, thanks for the patch!